batch2/batch/worker.py

```diff
@@ -28,19 +28,27 @@
 # uvloop.install()
 
 configure_logging()
-log = logging.getLogger('batch2-agent')
-
-docker = aiodocker.Docker()
+log = logging.getLogger('batch2-worker')
 
 MAX_IDLE_TIME_SECS = 60
 
+CORES = int(os.environ['CORES'])
 NAME = os.environ['NAME']
+NAMESPACE = os.environ['NAMESPACE']
 BUCKET_NAME = os.environ['BUCKET_NAME']
 INSTANCE_ID = os.environ['INSTANCE_ID']
+IP_ADDRESS = os.environ['IP_ADDRESS']
```

Comment:
I'd prefer the same order as they're set in the docker command. Also, why is the TOKEN not referenced here?

Reply:
I see it's inlined in the authorization header, I suppose you don't want to accidentally associate with the env vars that get logged?

Reply:
Yes, it's only referenced once when setting up the headers and it is "sensitive", so that seemed safer.

Reply:
Note, the token is getting logged in the startup section.